thirdparty/gflags/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(GFLAGS_IS_SUBPROJECT TRUE)
+set(HAVE_SYS_STAT_H 1)
+set(HAVE_INTTYPES_H 1)
+set(INTTYPES_FORMAT C99)
+set(BUILD_TESTING OFF)
+set(BUILD_SHARED_LIBS OFF)
+
+add_subdirectory(gflags EXCLUDE_FROM_ALL)
+set_target_properties(gflags_nothreads_static PROPERTIES FOLDER thirdparty)
+
+openvino_developer_export_targets(COMPONENT openvino_common TARGETS gflags)
```

Comment:
export must a part of `thirdparty/CMakeLists.txt` because this file is installed to iRC package where users don't have this function and don't have developer package

Reply:
Done

Reply:
Done